client/src/components/pages/admin/tableStyles.scss

```diff
@@ -0,0 +1,1533 @@
+
+$primary-color: #22313F;
+$gray-color: #ddd;
+$medium-up: 600px;
+$large-up: 1000px;
+$global-radius: 4px;
```

Comment:
This file looks like it includes dozens and dozens of style selectors that do not apply to the react component. It is also not being used in a way that keeps its effects scoped. A 1,500+ line CSS file is extremely excessive. There is also a blend of tab-indentations and spaces. Indentation should be in blocks of 4 spaces.

Please remove all unneeded styles from this file. It might be easier to just start over from scratch and copy in the styles you want to use instead of removing all the unneeded styles. 

Look into [CSS-Loader](https://github.com/webpack/css-loader#usage), which we are using. Think about how local scope can be used here.